sickbeard/providers/newznab.py

```diff
@@ -195,11 +200,11 @@ def _get_episode_search_strings(self, ep_obj, add_string=''):
                 params['q'] = helpers.sanitizeSceneName(cur_exception)
                 paramsNoEp = params.copy()
                 
-                paramsNoEp['q'] = paramsNoEp['q'] + " " + str(paramsNoEp['ep'])
-                if "ep" in paramsNoEp:
-                    paramsNoEp.pop("ep")
+                paramsNoEp['q'] = paramsNoEp['q'] + " " + str(paramsNoEp[episode])
```

Comment:
My understanding is that paramsNoEp['episode'] is already a string formated by "%i" % ... in line 171. Also worth noting that for both providers I've tried (usenet-crawler and nzb.su) having an episode without a 0 padding when when in single digits does not yield a result: q=Show.Name+3 => no result while q=Show.Name+03 has result